### PR TITLE
Update site for Core 5.0

### PR DESCRIPTION
**Pull request by volosied**
I created 3 commits in order to see the changes.

Please review.

### CODE REVIEW

File: core50.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+Implementation of the Jakarta Faces 5.0 specification.  This is still in development and not yet released. 
+
+## Requirements
+* Java 21
+* Servlet 6.1+
+* EL 6.0
+* CDI 4.1
+* JSTL 3.0 (optional)
+* BV 3.1 (optional)
```
Comment (volosied) [inline core50.md:11]: Updated to EE11. I believe 21 is the required runtime based on my current understanding.

**Review by melloware — APPROVED** [review]
LGTM but yeah @tandraschko might know the minimum for 5.0

**Comment by tandraschko** [comment]
our minimum doesnt change actually, maybe we can write something like Java XX (EE Plattform: 21)

File: core50.md (hunk 1 above)
Comment (volosied) [inline core50.md:11]:
```suggestion
* Java  11 (EE Platform: 21) 
* Servlet 6.0 (EE Platform: 6.1)
* EL 5.0 (EE Platform: 6.0)
* CDI 4.0 (EE Platform: 4.1)
* Tags (EE Platform: 3.0) (optional)
* BeanValidation 3.0 (EE Platform: 3.1) (optional)
```

Comment (volosied) [reply core50.md:11]: @tandraschko  Something like this?  Perhaps we should use Java 17, but I can start a mailing list discussion about this.

Comment (volosied) [reply core50.md:11]: @tandraschko Can you comment? See the emails here: https://www.eclipse.org/lists/faces-dev/msg00354.html

There's a push to move to 17 or 21.

Comment (tandraschko) [reply core50.md:11]: +0 tbh
As long we dont need to up, we can simply stay on java11

**Comment by tandraschko** [comment]
@volosied i would set java to 17 and then merge pls

**Pull request merged**